registry/connect/views.py

```diff
@@ -37,9 +37,9 @@ def docker():
     sources = get_sources(user_info)
 
     install_commands = {
-        source: "docker run -v $PWD/tokens:/etc/condor/tokens.d opensciencegrid/open-science-pool-registry:fresh register.py --local-dir $PWD/tokens --host {}".format(
-            source
-        )
+        source: "mkdir tokens && \\" +
```

Comment:
```suggestion
        source: "mkdir -p tokens && \\" +
```
without the -p, mkdir errors if the dir already exists.